components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/BrowserAction.kt

```diff
@@ -25,4 +25,21 @@ data class BrowserAction(
     val badgeTextColor: Int?,
     val badgeBackgroundColor: Int?,
     val onClick: () -> Unit
-)
+) {
+    /**
+     * Return a copy of [BrowserAction] with tab-specific [override] if it exists.
```

Comment:
nit: `Returns a copy of this [BrowserAction] with the provided overrides applied e.g. for tab-specific overrides`.